src/graphics/clouds.c

```diff
@@ -269,8 +271,16 @@ void clouds_draw(int x_offset, int y_offset, int x_limit, int y_limit, float bas
             cloud->status = STATUS_INACTIVE;
             continue;
         }
-        speed_set_target(&cloud->speed.x, -CLOUD_SPEED, SPEED_CHANGE_IMMEDIATE, 1);
-        speed_set_target(&cloud->speed.y, CLOUD_SPEED / 2, SPEED_CHANGE_IMMEDIATE, 1);
+
+        double cloudSpeed = 0;
```

Comment:
```suggestion
        double cloud_speed = 0;
```

Proper naming convention please!